es/es-testing/src/main/java/org/elasticsearch/cluster/MockInternalClusterInfoService.java

```diff
@@ -96,6 +96,9 @@ public void onFailure(Exception e) {
 
     private List<NodeStats> adjustNodesStats(List<NodeStats> nodesStats) {
         BiFunction<DiscoveryNode, FsInfo.Path, FsInfo.Path> diskUsageFunction = this.diskUsageFunction;
+        if (diskUsageFunction == null) {
```

Comment:
Kind of dropped it before as well while removing the `adjustNodesStats` from production code.
See https://github.com/crate/crate/pull/9153#discussion_r327588628